swagger.yaml

```diff
@@ -39,23 +127,160 @@ parameters:
+            description: Default latitude to go to when selecting route on map
+          longitude:
+            type: number
+            format: float
+            description: Default longitude to go to when selecting route on map
+          zoomLevel:
+            type: integer
+            description: Default zoom level to use when selecting route on map
+          stops:
+            type: array
+            description: Locations where the bus stops along the route
+            items:
+              properties:
+                stopID:
+                  type: string
+                description:
+                  type: string
+                latitude:
+                  type: number
+                  format: float
+                  description: Latitude of stop
+                longitude:
+                  type: number
+                  format: float
+                  description: Longitude of stop
+                order:
+                  type: integer
+                  description: The order this stop is used relative to the other stops
+  VehiclesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/VehicleResourceObject"
+  VehicleResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/VehicleResourceObject"
+  VehicleResourceObject:
+    properties:
+      id:
+        type: string
+        description: vehicleID
+      type:
+        type: string
+        description: route
+      attributes:
+        properties:
+          routeID:
+            type: string
+            description: routeID for route of vehicle
+          name:
+            type: string
+            description: Name of vehicle
+          latitude:
+            type: number
+            format: float
+            description: Latitude of vehicle's position
+          longitude:
+            type: number
+            format: float
+            description: Longitude of vehicle's position
+          speed:
+            type: number
+            format: float
+            description: Speed of vehicle (MPH)
+          heading:
+            type: integer
+            description: Compass heading of vehicle (0-359)
+          age:
+            type: integer
+            description: Seconds since vehicle position was updated
+          onRoute:
+            type: boolean
+            description: Whether the vehicle is on its route
+          delayed:
+            type: boolean
+            description: Whether the vehicle is delayed
+  ArrivalsResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResourceObject:
+    properties:
+      id:
         type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
+        description: arrivalID
```

Comment:
The vendor doesn't give any unique ID for a arrival. This will have to be a concatenation or hash of routeID and stopID.

Reply:
What's the nature of an arrival? It seems like they are transitory, whereas routes and vehicles are relatively permanent. So does it make sense for an arrival to have a permanent ID? Do we want to re-use old arrival IDs or not?

Reply:
I am confused about arrival ID too. If the arrival ID is the concatenation of route ID and stop ID, will we get a different response if request /arrivals?stopID=xxx&routeID=yyy ?

Reply:
Calling `/arrivals?stopID=xxx&routeID=yyy` would return a single arrival, since filtering by both stopID and routeID would only produce one arrival.

Reply:
I was just trying to think of what to assign to the ID field as per JSON API spec. Arrivals don't exist permanently - An arrival will only exist if something is arriving. Does it make sense to have an ID for something that won't always exist? Should we combine arrivals into one of the other endpoints?

Reply:
@tsoliangwu0130 and I discussed the `/arrivals/{arrivalID}` endpoint and we agreed it doesn't have much benefit. I think we still should provide a unique ID for each arrival resource though per [JSON API spec](http://jsonapi.org/format/#document-resource-object-identification).